gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Let me remind that Gradle wrapper should be updated with `gradlew wrapper --gradle-version=8.6` instead of manual change of Gradle wrapper property.

Reply:
Because the command updates not only `gradle/wrapper/gradle-wrapper.properties`, but `gradle/wrapper/gradle-wrapper.jar`, `gradlew`, and `gradlew.bat` as well, which are not updated in the PR. Also, it is [the recommended way](https://docs.gradle.org/8.6/release-notes.html#upgrade-instructions) to upgrade your wrapper.